### PR TITLE
Small fixes and add tools to plugins

### PR DESCRIPTION
**Pull request by d1nuc0m**
- Fix puppet-ghostbusters URL in Plugins
- Add to plugins:
    - g10k
    - Hiera Data Manager
    - Puppet Summary

### CODE REVIEW

File: _data/tools/hdm.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "hdm",
+    "display_name": "Hiera Data Manager",
+    "description": "Web interface for Hiera data",
+    "url": "https://github.com/betadots/hdm"
+}
```
Comment (bastelfreak) [inline _data/tools/hdm.json:6]: Can you please add the missing \n?

File: _data/tools/puppet-summary.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "puppet-summary",
+    "display_name": "Puppet Summary",
+    "description": "Another web interface for Puppet reports",
+    "url": "https://github.com/skx/puppet-summary"
+  }
```
Comment (bastelfreak) [inline _data/tools/puppet-summary.json:6]: same here

Comment (d1nuc0m) [reply _data/tools/puppet-summary.json]: Done, had to force push to verify previous commits

File: _data/tools/hdm.json (hunk 1 above)
Comment (d1nuc0m) [reply _data/tools/hdm.json]: Done, had to force push to verify sign commits

**Pull request merged**